NAMESPACE

```diff
@@ -76,8 +76,10 @@ export(font_collection)
 export(font_face)
 export(font_google)
 export(font_link)
+export(icon_spinner)
```

Comment:
I agree w/ @gadenbuie that we shouldn't export this (but use it internally for a default value)